testing/go/enginetest/doltgres_engine_test.go

```diff
@@ -171,23 +171,31 @@ func TestSingleScript(t *testing.T) {
 
 	var scripts = []queries.ScriptTest{
 		{
-			Name: "people tables",
+			Name: "topN stable output",
 			SetUpScript: []string{
-				"CREATE TABLE `people` (   `dob` date NOT NULL," +
-					"   `first_name` varchar(20) NOT NULL," +
-					"   `last_name` varchar(20) NOT NULL," +
-					"   `middle_name` varchar(20) NOT NULL," +
-					"   `height_inches` bigint NOT NULL," +
-					"   `gender` bigint NOT NULL," +
-					"   PRIMARY KEY (`dob`,`first_name`,`last_name`,`middle_name`) )",
-				`insert into people values ('1970-12-1'::date, 'jon', 'smith', 'a', 72, 0)`,
+				"create table xy (x int primary key, y int)",
+				"insert into xy values (1,0),(2,0),(3,0),(4,0)",
 			},
 			Assertions: []queries.ScriptTestAssertion{
 				{
-					Query: "select * from people order by dob",
-					Expected: []sql.Row{
-						{"1970-12-01", "jon", "smith", "a", int64(72), int64(0)},
-					},
+					Query:    "select * from xy order by y asc limit 1 offset 1",
+					Expected: []sql.Row{{2, 0}},
+				},
+				{
+					Query:    "select * from xy order by y asc limit 1 offset 2",
+					Expected: []sql.Row{{3, 0}},
+				},
+				{
+					Query:    "select * from xy order by y asc limit 1 offset 3",
+					Expected: []sql.Row{{4, 0}},
+				},
+				{
+					Query:    "(select * from xy order by y asc limit 1 offset 1) union (select * from xy order by y asc limit 1 offset 2)",
+					Expected: []sql.Row{{2, 0}, {3, 0}},
+				},
+				{
+					Query:    "with recursive cte as ((select * from xy order by y asc limit 1 offset 1) union (select * from xy order by y asc limit 1 offset 2)) select * from cte",
```

Comment:
I'm not seeing these tests being added to our _actual_ tests anywhere.

In GMS we used this `TestSingleScript` workflow because we didn't have anything better, but we _do_ have something better in Doltgres with the `Focus` field on scripts. That way, you could write these in `SmokeTests` (or wherever) to test your logic, and throw on that `Focus: true` flag so that only these run. In effect you get the same experience here. But then when you're done, delete the flag, and now they're actual tests in our suite (with the added benefit of CI protection in case you forget to remove the flag). Easy way to expand our test coverage since you don't have to copy the tests out later, we're writing new tests directly into our normal test structures.

Reply:
I was missing these originally but they're in there now.